### PR TITLE
Release notes for 5.7.26-29

### PR DESCRIPTION
**Pull request by hrvojem**

### CODE REVIEW

File: doc/source/release-notes/Percona-Server-5.7.26-29.rst [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+Bugs Fixed
+==========
+
+- TokuDB storage engine would assert on load when used with jemalloc 5.x.
+  Bug fixed :psbug:`5406`
+
+- a read-write workload on compressed InnoDB tables could cause an assertion
+  error. Bug fixed :psbug:`3581`.
+	
+- using TokuDB or MyRocks native partitioning and ``index_merge`` could lead to
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.7.26-29.rst]:
```suggestion
- using TokuDB or MyRocks native partitioning and ``index_merge`` access method could lead to
```

File: doc/source/release-notes/Percona-Server-5.7.26-29.rst [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+- a stack buffer overrun could happen if the redo log encryption with
+  key rotation was enabled. Bug fixed :psbug:`5305`.
+
+- TokuDB and MyRocks native partitioning handler objects were allocated from a 
+  wrong memory allocator. Memory was released only on shutdown and concurrent
+  access to global memory allocator caused memory corruptions and therefore
+  crashes. Bugs fixed :psbug:`5508`, :psbug:`5525`.
+
+- redo log encryption wasn't working. Bug fixed :psbug:`5547`.
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.7.26-29.rst]: Needs more details. Enabling redo log encryption resulted in redo log being written unencrypted or similar

File: doc/source/release-notes/Percona-Server-5.7.26-29.rst [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  intermediate binary log server in a replication chain. Bug fixed
+  :psbug:`5353` (upstream :mysqlbug:`93917`).
+
+- when :ref:`audit_log_plugin` was enabled, the server could use a lot of memory when
+  handling large queries.  Bug fixed :psbug:`5395`.
+
+- :ref:`changed_page_tracking` was missing pages changed by the in-place DDL.
+  Bug fixed :psbug:`5447`.
+
+- :variable:`innodb_encrypt_tables` variable accepted ``FORCE`` option only
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.7.26-29.rst:73]: Not clear enough - what is FORCE as not-string?

File: doc/source/release-notes/Percona-Server-5.7.26-29.rst [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+  as a string. Bug fixed :psbug:`5538`.
+
+- Redo log encryption would fail to decrypt pages with a certain bit set
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.7.26-29.rst]: Enabling redo log encryption and changed page tracking together would result in the error log flooded with decryption errors.

File: doc/source/release-notes/Percona-Server-5.7.26-29.rst [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+  when :ref:`changed_page_tracking` was enabled. While the end result is
+  the same, the error log gets flooded with decryption errors.
+  Bug fixed :psbug:`5541`.
+
+- system keyring keys initialization wasn't thread safe. Bugs fixed
+  :psbug:`5554`.
+
+- Docker image is now ignoring password that is set in the configuration
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.7.26-29.rst]: Please get feedback from build team

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: doc/source/release-notes/Percona-Server-5.7.26-29.rst (hunk 5 above)
Comment (EvgeniyPatlan) [reply doc/source/release-notes/Percona-Server-5.7.26-29.rst]: @hrvojem the description can be taken from https://github.com/percona/percona-docker/pull/87
Also if you set password once starting container using -e MYSQL_ROOT_PASSWORD and you will mount your own .cnf but these two password are different - password from MYSQL_ROOT_PASSWORD will be used

**Review by laurynas-biveinis — APPROVED** [review]
OK once 5573 description is OK

**Review by EvgeniyPatlan — APPROVED** [review]

**Pull request merged**